src/index.ts

```diff
@@ -213,15 +218,15 @@ async function addResolverOutput(
 	additionalTemplateVars?: Dictionary<string>,
 ): Promise<void> {
 	// Now read the file, allow the resolver to process it, and return it
-	const extraFiles = await resolver.resolve(
+	const dockerfile = await resolver.resolve(
 		bundle,
 		specifiedDockerfilePath,
 		additionalTemplateVars,
 	);
 
-	for (const file of extraFiles) {
-		pack.entry({ name: file.name, size: file.size }, file.contents);
-	}
+	const content = dockerfile.contents.toString();
+	const newContent = (await bundle.callDockerfileHook(content)) || content;
+	pack.entry({ name: dockerfile.name, size: content.length }, newContent);
```

Comment:
`content.length` → `newContent.length`

Reply:
Good catch! It seems to not make a difference as the tests still passed, but better to be precise